apps/evm/src/clients/subgraph/queries/governance/getProposals/index.ts

```diff
@@ -0,0 +1,14 @@
+import { request } from 'graphql-request';
+
+import { ProposalsDocument } from 'clients/subgraph/gql';
+import { SUBGRAPH_URlS } from 'constants/subgraphUrls';
+import type { ChainId } from 'types';
+
+export interface GetProposalsInput {
+  chainId: ChainId;
+}
+
+export const getProposals = ({ chainId }: GetProposalsInput) =>
+  SUBGRAPH_URlS.governance[chainId]
+    ? request(SUBGRAPH_URlS.governance[chainId]!, ProposalsDocument)
+    : undefined;
```

Comment:
I've a question: is the caller of this function going to pass either BSC_MAINNET or BSC_TESTNET? If I got it right, if we are in testnet we always reach out for the BSC_TESTNET subgraph, no matter if the app is using another chain

Reply:
Understood, I was checking if you wanted to make this function be the one to select either the BSC_TESTNET or the BSC_MAINNET subgraph based on the environment rather than the chain passed